build.gradle

```diff
@@ -34,35 +34,33 @@ allprojects {
     }
 
     repositories {
-        ['TBD'].each { repo ->
-            maven {
-                credentials {
-                    username = findProperty('user') ?: System.getenv('user')
-                    password = findProperty('password') ?: System.getenv('password')
-                }
-
-                url "TBD/$repo/"
+        maven {
+            credentials {
+                username = findProperty('ACCESS_TOKEN_USER') ?: System.getenv('ACCESS_TOKEN_USER')
+                password = findProperty('ACCESS_TOKEN_VALUE') ?: System.getenv('ACCESS_TOKEN_VALUE')
```

Comment:
Can we make these values consistent across our Gradle projects?

Reply:
So far I think most of us are using `GITHUB_ACTOR` and `GITHUB_TOKEN`